### DISCUSSION TITLE
Http problem 403 forbidden ports 5 number

### DISCUSSION THREAD

**Discussion by Raffy142** [xjasonlyu/tun2socks#420] [Q&A]
Hi, i have problem with proxy 5 ports, when i set i get this error
Can someone help me?

time="2024-10-16T23:26:35Z" level=info msg="[DIALER] set fwmark: 0x22b"
time="2024-10-16T23:26:35Z" level=info msg="[STACK] tun://tun0 <-> http://216.146.20.92:25971/"
time="2024-10-16T23:26:37Z" level=info msg="[TCP] 198.18.0.1:46118 <-> 104.21.87.17:443"
time="2024-10-16T23:26:38Z" level=warning msg="[TCP] dial 208.95.112.1:80: HTTP connect status: 403 Forbidden"
time="2024-10-16T23:26:38Z" level=info msg="[TCP] 198.18.0.1:41806 <-> 172.67.139.50:443"
time="2024-10-16T23:26:39Z" level=info msg="[TCP] 198.18.0.1:46130 <-> 104.21.87.17:443"
time="2024-10-16T23:26:39Z" level=warning msg="[TCP] dial 207.246.122.60:7070: HTTP connect status: 403 Forbidden"
time="2024-10-16T23:26:40Z" level=warning msg="[TCP] dial 207.246.122.60:7070: HTTP connect status: 403 Forbidden"
time="2024-10-16T23:26:41Z" level=warning msg="[TCP] dial 207.246.122.60:7070: HTTP connect status: 403 Forbidden"

**Comment by xjasonlyu** [comment]
Please check your proxy server. It looks like a server side issue.
